TrackletAlgorithm/Constants.h

```diff
@@ -4,8 +4,7 @@
 #include "ap_int.h"
 
 #ifdef CMSSW_GIT_HASH
-  #include "Settings.h"
```

Comment:
According to the [CMSSW style guide](http://cms-sw.github.io/cms_coding_rules.html#3--style-rules-1) please move all preprocessor directives to the left margin.